File: copy_logos.js
const fs = require('fs');
const path = require('path');

const srcDir = 'C:\\Users\\User\\Desktop\\Sites\\brincakids-landing-page\\assets';
const dstDir = 'C:\\Users\\User\\.gemini\\antigravity-ide\\scratch\\brincakids-landing-page\\assets';

try {
    const files = fs.readdirSync(srcDir);
    console.log("Files in src:", files);

    files.forEach(f => {
        if (f.includes('3') && f.endsWith('.png')) {
            fs.copyFileSync(path.join(srcDir, f), path.join(dstDir, 'logo.png'));
            fs.copyFileSync(path.join(srcDir, f), path.join(srcDir, 'logo.png'));
            console.log("Copied transparent logo:", f);
        } else if (f.includes('2') && f.endsWith('.png')) {
            fs.copyFileSync(path.join(srcDir, f), path.join(dstDir, 'logo-gradient.png'));
            fs.copyFileSync(path.join(srcDir, f), path.join(srcDir, 'logo-gradient.png'));
            console.log("Copied gradient logo:", f);
        }
    });
} catch (e) {
    console.error("Error:", e.message);
}

File: copy_logos.py
import shutil
import os

src_dir = r"C:\Users\User\Desktop\Sites\brincakids-landing-page\assets"
dst_dir = r"C:\Users\User\.gemini\antigravity-ide\scratch\brincakids-landing-page\assets"

try:
    files = os.listdir(src_dir)
    print("Files in src:", files)

    for f in files:
        if "3" in f and f.endswith(".png"):
            shutil.copy(os.path.join(src_dir, f), os.path.join(dst_dir, "logo.png"))
            shutil.copy(os.path.join(src_dir, f), os.path.join(src_dir, "logo.png"))
            print("Copied transparent logo:", f)
        elif "2" in f and f.endswith(".png"):
            shutil.copy(os.path.join(src_dir, f), os.path.join(dst_dir, "logo-gradient.png"))
            shutil.copy(os.path.join(src_dir, f), os.path.join(src_dir, "logo-gradient.png"))
            print("Copied gradient logo:", f)
except Exception as e:
    print("Error:", e)
